travis/Dockerfile

```diff
@@ -5,7 +5,14 @@ FROM ruby:latest
 # and install dependencies
 RUN apt-get update \
     && apt-get -y install maven \
-    && apt-get -y install python3-requests 
+    && apt-get -y install python3-requests \
+    # Install Chrome for Selenium
+    && curl https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o /chrome.deb \ 
+    # Install chromedriver for Selenium
+    && dpkg -i /chrome.deb || apt-get install -yf \
+    && curl https://chromedriver.storage.googleapis.com/2.31/chromedriver_linux64.zip -o /usr/local/bin/chromedriver \
```

Comment:
I thought we didn't need to install the drive since the `io.github.bonigarcia.wdm.WebDriverManager;` library manages that for us.